### ISSUE TITLE
Automatic docker login fails

### ISSUE THREAD

**Issue by mvz** [getkuby/kuby-core#37]
When not logged in with the Docker registry, attempting to push gives me this result:

```bash
$ bundle exec kuby -e production push
Attempting to log in to registry at docker.pkg.github.com
Error: Cannot perform an interactive login from a non TTY device
Couldn't log in to the registry at docker.pkg.github.com
build failed: docker command exited with status code 1
```

If I log in by hand with the same user and password, it works fine and the push succeeds.

**Comment by mvz** [comment]
I think this actually has to do with dotenv not getting loaded in my setup.

**Comment by mvz** [comment]
Indeed, that was the problem. I'm using `dotenv` instead of Rails' encrypted storage and had to put `require "dotenv/load"` at the top of my `kuby.rb` file to make the environment variables load.

**Comment by camertron** [comment]
@mvz ah yeah that makes sense, glad you got things sorted out :)

**Comment by ibrahima** [comment]
Hi! I'm getting the same error. I'm probably not following the instructions exactly, since I'm just trying to get started learning K8s/Kuby with Kind on my laptop. I then did a `docker login` and am still getting the same error - is that expected? Should I have provided credentials via encrypted credentials or env vars in kuby.rb even though I have run `docker login`? Thanks! Sorry for bumping an old issue, I just wanted to keep the discussion in a related place in case I'm doing something wrong.
